### DISCUSSION TITLE
What is the difference between camera => history and reviews?

### DISCUSSION THREAD

**Discussion by ovizii** [blakeblackshear/frigate#12877] [General]
Both look similar and yet different. What exactly is each purpose and why do we need both? Both seem to lead to the reviews menu icon on the left.

the history part is filterable and has 2 views: timeline and events.

![image](https://github.com/user-attachments/assets/d9e48332-22f1-4385-9f6d-405dbf7de6cf)

The reviews section just looks slightly different:

![image](https://github.com/user-attachments/assets/edb421d6-4a5e-4291-8fe8-167a97c2d016)

**Comment by NickM-27** [comment]
history is where you view full res recordings and can scrub the timelne, switching between cameras. Reviews is where you can do a first level review with the review items. It is easy to see an overview of the activity on your camera and you can hover (desktop) or swipe right (mobile) to view the previews for each review item. Making it less necessary to view the recordings unless something requires a closer look

**Comment by hawkeye217** [comment]
A user asked about this the other day: https://github.com/blakeblackshear/frigate/discussions/12801

The details are explained in the [release notes](https://github.com/blakeblackshear/frigate/releases/tag/v0.14.0).

> Review
> The Review area of Frigate 0.14 is for quickly reviewing historical footage of interest from your cameras. Review items are indicated on a vertical timeline and displayed as a grid of previews - bandwidth-optimized, low frame rate, low resolution videos. Hovering over or swiping a preview plays the video and marks it as reviewed. If more in-depth analysis is required, the preview can be clicked/tapped and the full frame rate, full resolution recording is displayed. Review items are filterable by date, object type, and camera.

> History
> In Frigate 0.14, full frame rate, full resolution camera footage (called "recordings" in earlier versions) can be viewed and scrubbed via a timeline. Previews of your other cameras are displayed and scrubbed alongside the main selected camera.

**Comment by ovizii** [comment]
Thanks for clarifying and the pointer to the otehr issue. Sorry for the duplciate question, I didn't see the other one.

**Reply by hawkeye217** [reply]
No worries. Happy to help.
